### PR TITLE
Patch resprite

### PR DESCRIPTION
**Pull request by Xander3359**
## About The Pull Request
Adds more selectable patches.
image

## Why It's Good For The Game
Currently there are only 4 patches to pick from. This selection is tragic, so how about a few more?
Adds more patches so that chemists have some choice.
## Changelog
:cl: @Xander3359 Sprites: Nebby#1467
image: Replaces the old patches with some new sprites
/:cl:

### CODE REVIEW

**Comment by CRITAWAKETS** [comment]
screenshots plz

also im not entirely sure about the chaos patches

**Comment by Jolly-66** [comment]
You also deleted something, don't know if its intentional or not

image

**Comment by Xander3359** [comment]
> 

how the fuck

**Comment by Xander3359** [comment]
image
This is probably the best way to view them, through icondiff bot

**Comment by Elli-Skala** [comment]
> image This is probably the best way to view them, through icondiff bot

bandaid_exclaim and bandaid_exclaimtwice apper to share the same sprite?

**Comment by moocowswag** [comment]
They both have two !

**Comment by MTandi** [comment]
image

Make the skull fit that 5x5 inner area, this doesn't look like a patch anymore.

image
image

These color choices hurts eyes 

image

image

Looks like a clump of pixels. Is there some meaning behind it? I think it's better to color code it and add some letters instead of random pixel shapes.

image

image

I would not understand that this is supposed to be exclamation marks, if I didn't see the filename.

**Comment by MTandi** [comment]
Maybe make them in colors of medkits and add some more recognizable shapes

image

**Comment by Cheshify** [comment]
Please keep the Chaos patches, the homies and I must hit those Tzeench Space Drugs

**Comment by TheVekter** [comment]
bandaid_firstaid can't have a red cross on it, that's a WAR CRIME, XANDER

**Comment by Xander3359** [comment]
After feedback, this is what we ended up with.
image

**Comment by Wallemations** [comment]
I'll be honest those numbers look really bad. You might want to work on those a bit more, MTandi showed some in a cleaner font that would be nice.

**Comment by MTandi** [comment]
You can use https://github.com/tgstation/tgstation/blob/master/icons/misc/Font_Minimal.dmi which has 3x5 typeset characters

**Comment by Cheshify** [comment]
> Sweating as the virologist begins slapping nurgle patches, honestly keep em its a small refrence and is kinda funny

Virology nurgle patches make this PR an essential merge

**Comment by jehnicat** [comment]
> > Sweating as the virologist begins slapping nurgle patches, honestly keep em its a small refrence and is kinda funny
> 
> Virology nurgle patches make this PR an essential merge

Hi spriter here no chaos patches; maybe in another PR. I've already made Xander do too much for me l0l.

**Comment by Cheshify** [comment]
JEHNI NOOO

**Comment by CRITAWAKETS** [comment]
By the way for the chaos patches, the names might be a bit litigious because of how geedubs is, so i'd recommend instead using alternate names. If you want a good reference, use the names from _Stellaris_ which correspond to all the chaos gods.

File: code/__DEFINES/reagents.dm [hunk 1]
```diff
@@ -47,8 +47,30 @@
...
+	"bandaid_3", \
+	"bandaid_4", \
+	"bandaid_blank", \
+	"bandaid_both", \
+	"bandaid_brute", \
+	"bandaid_brute_2", \
+	"bandaid_burn", \
+	"bandaid_burn_2", \
+	"bandaid_clown", \
+	"bandaid_colonthree", \
```
Comment (JohnFulpWillard) [inline code/__DEFINES/reagents.dm:62]: this is the worst thing i've ever seen

Comment (Xander3359) [reply code/__DEFINES/reagents.dm:62]: I was told this is the proper way to make define lists

Comment (Mothblocks) [reply code/__DEFINES/reagents.dm:62]: Define lists are fundamentally not a good idea. You didn't make this code so you don't have to fix it but the usual proper way to make define lists is not at all

**Review by Mothblocks — APPROVED** [review]

File: code/__DEFINES/reagents.dm (hunk 1 above)
Comment (Fikou) [reply code/__DEFINES/reagents.dm:62]: i think he means the :3 sticker

**Review by Fikou — CHANGES_REQUESTED** [review]
fix your changelog, `image` is not a valid entry

**Comment by Xander3359** [comment]
image
Whats the entry for sprites then?
I used image: , because that's what is in the template.

**Comment by Mothblocks** [comment]
@Fikou i thought tattle combined imageadd and imagedel

**Comment by Mothblocks** [comment]
https://github.com/tgstation/tgstation/pull/74865

yeap

**Pull request merged**